process/consensus.md

```diff
@@ -87,20 +91,23 @@ We differentiate the following cases:
    three contributors of different affiliations is acceptable as long as there
    is no objection. Consensus will be deemed to not have been reached if
    interested parties did not sign off. At any point in time a contributor can
-   request that final consensus be delayed to an in-person meeting. In this
+   request that final consensus be delayed to a subgroup or CG meeting. In this
    case, the chair puts the item on the group's agenda of upcoming
    discussions.
 3. Substantial technical changes or additions are usually carried in their own
    GitHub repository by a champion. It is critical that these proposals be able
    to evolve quickly without much process. Early on in such a proposal's
    lifetime no consensus is needed and a single champion can modify the proposal
    at will. As the proposal matures it is expected that the champion will seek
-   collaborators to form a small group. Gauging consensus in the small group is
-   left up to the champion, with input from the chair. When a proposal is near
-   maturity the champion shall bring it to a meeting and seek wider
-   consensus on open design points and contended issues. All decisions made by
-   the small group can be revisited until consensus is reached at an in-person
-   meeting.
+   collaborators to form a small group, or use the subgroup if one relevant to the
+   proposal exists. Gauging online consensus in the small group is left up to the
+   champion or in some cases the subgroup chair. Gauging consensus at a subgroup meeting is
```

Comment:
"in some cases" still seems underspecified. Should we just say that either the champion or relevant subgroup chair can declare consensus for online discussions?

Reply:
I think the issue is that 'small groups' are not always subgroups, maybe it would be better to say that champion is responsible if it is a subgroup (subgroup maybe takes precedence over 'small group'). TBH, I don't know if we really use 'small group' concept in practice.